integration-tests/test.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+set +x
+set -e
+
+
+# TODO: Add logic for running with all
+REPO=$1
+TESTDATA=testdata
+
+ALLOWED_FAILURE_REPOS=("geb" "drai" "guni-lev")
+if [[ " ${ALLOWED_FAILURE_REPOS[*]} " =~ " ${REPO} " ]]; then
+    export FORGE_ALLOW_FAILURE=1
+fi
+
+DIR=`pwd`
+
+function runTests() {
+    cd $TESTDATA/$1
+
+    # run any installation step if needed
+    make install || true
+
+    # update the deps
+    $DIR/../target/debug/forge update
```

Comment:
Given we're benchmarking, should we do a release build instead and call the release binary? Would also be nice if the binary could itself be part of an env var so that running the tests locally becomes easy